website/docs/docs/build/metrics.md

```diff
@@ -324,6 +345,66 @@ Note that `value` must be defined as a string in YAML, because it will be compil
         value: "'2020-01-01'"
 ```
 
+### Calendar
+The dbt_metrics package contains a [basic calendar table](https://github.com/dbt-labs/dbt_metrics/blob/main/models/dbt_metrics_default_calendar.sql) that is created as part of your `dbt run`. It contains dates between 2010-01-01 and 2029-12-31. 
+
+If you want to use a custom calendar, you can replace the default with any table which meets the following requirements:
+- Contains a `date_day` column. 
+- Contains the following columns: `date_week`, `date_month`, `date_quarter`, `date_year`, or equivalents. 
+- Additional date columns need to be prefixed with `date_`, e.g. `date_4_5_4_month` for a 4-5-4 retail calendar date set. Dimensions can have any name (see [dimensions on calendar tables](#dimensions-on-calendar-tables)).
+
+To do this, set the value of the `dbt_metrics_calendar_model` variable in your `dbt_project.yml` file: 
+```yaml
+#dbt_project.yml
+config-version: 2
+[...]
+vars:
+    dbt_metrics_calendar_model: my_custom_calendar
+```
+
+<VersionBlock firstVersion="1.3">
+
+### Configuration 
+
+Metric nodes now accept `config` dictionaries like other dbt resources. Metric configs can specified in the metric yml itself or for groups of metrics in the `dbt_project.yml` file.
+
+```yml
+# in metrics.yml
+version: 2
+
+metrics:
+  - name: config_metric
+    label: Example Metric with Config
+    model: ref('my_model')
+    calculation_method: count
+    timestamp: date_field
+    time_grains: [day, week, month]
+
+    config:
+      enabled: True
+```
+
+Or:
+
+```yml
+# in dbt_project.yml
+
+metrics: 
+  your_project_name: 
+    +enabled: true
+```
+
+#### Accepted Metric Configurations
+
+Below is the list of metric configs currently accepted.
+
+| Config | Type | Accepted Values | Default Value | Description |
+|--------|------|-----------------|---------------|-------------|
+| `enabled` | boolean | True/False | True | Enables or disables a metric node. When disabled, dbt will not consider it as part of your project. |
+| `treat_null_values_as_zero` | boolean | True/False | True | Controls the `coalesce` behavior for metrics. By default, when there are no observations for a metric, the output of the metric as well as period Over period secondary calculations will include a `coalesce({{ field }}, 0)` to return 0's rather than nulls. Setting this config to False instead returns `NULL` values. |
```

Comment:
suggesting link to period over period for context

```suggestion
| `treat_null_values_as_zero` | boolean | True/False | True | Controls the `coalesce` behavior for metrics. By default, when there are no observations for a metric, the output of the metric as well as [Period over Period](https://github.com/dbt-labs/dbt_metrics#period-over-period-source) secondary calculations will include a `coalesce({{ field }}, 0)` to return 0's rather than nulls. Setting this config to False instead returns `NULL` values. |
```